test/Interop/Cxx/class/nonescapable-lifetimebound.swift

```diff
@@ -1,6 +1,6 @@
 // RUN: rm -rf %t
 // RUN: split-file %s %t
-// RUN: %target-swift-frontend -I %swift_src_root/lib/ClangImporter/SwiftBridging  -I %t/Inputs -emit-sil %t/test.swift -enable-experimental-feature NonescapableTypes -cxx-interoperability-mode=default -diagnostic-style llvm 2>&1 | %FileCheck %s
+// RUN: %target-swift-frontend -I %swift_src_root/lib/ClangImporter/SwiftBridging  -I %t/Inputs -emit-silgen %t/test.swift -enable-experimental-feature NonescapableTypes -cxx-interoperability-mode=default -diagnostic-style llvm 2>&1 | %FileCheck %s
```

Comment:
I switched to `silgen` from `sil` because some SIL passes still crash on the SIL that we generate. This way I can avoid running the problematic passes and still test that we generate the SIL we want. That being said, we should switch back to `sil` as soon as the underlying problem is fixed (which should be soonish).